spec/models/legislator_spec.rb

```diff
@@ -34,6 +34,31 @@
 require 'rails_helper'
 
 describe Legislator do
+  describe ".create_or_update" do
+    context "new record" do
+      it "creates record with appropriate values" do
+        FactoryGirl.create(:state, abbrev: 'CA')
+        hash = { bioguide_id:  'A001',
+                 chamber:      'senate',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.